### ISSUE TITLE
[Bug] Contact Birthdays - Incorrect age for contacts with missing birthday year or year before 1970

### ISSUE THREAD

**Issue by lfom** [SimpleMobileTools/Simple-Calendar#2057]
Hello, there.

Thank you for SimpleMobileTools. Simple Calendar has a problem with the private contacts birthdays: for contacts older than 53 years or without a birthday year, the app will always show them as having 53 years old. Probably there is some problem when calculating the age in the calendar, as the Contacts/Dialer app show the ages correctly. Of course, the same problem happens in the widget.

To summarize, there are two problems:

1. Wrong age for contacts born before 1970
2. When a contact does not have a birthday year it is also displayed as having 53yo (i.e. the app uses 1970 as birthday year).

You may want to check #1812 for more info/screenshots.

SimpleCalendar 6.21.7 Pro (F-Droid)
RAV - Android 11 - HavocOS

Regards

**Comment by tibbi** [comment]
duplicate of #1812
